### PR TITLE
chore: update angular to 18

### PR DESCRIPTION
**Pull request by AtofStryker**
- updates the standalone app to angular 18
- adds @cypress/angular-signals test harness examples to test angular components with signals

### CODE REVIEW

File: angular-standalone/package.json [hunk 1]
```diff
@@ -7,36 +7,36 @@
...
-    "@types/jasmine": "~4.0.0",
-    "cypress": "^12.14.0",
-    "jasmine-core": "~4.1.0",
-    "karma": "~6.3.0",
-    "karma-chrome-launcher": "~3.1.0",
+    "@angular-devkit/build-angular": "^18.0.2",
+    "@angular/cli": "^18.0.2",
+    "@angular/compiler-cli": "^18.0.0",
+    "@types/jasmine": "~5.1.0",
+    "cypress": "https://cdn.cypress.io/beta/npm/13.12.0/darwin-arm64/angular-signals-ct-harness-c6cba8b03a7d7d098cdb469919eb674d40277604/cypress.tgz",
```
Comment (jordanpowell88) [inline angular-standalone/package.json:32]: We should try to remember to go and change this after it's released

File: angular-standalone/src/app/app.component.spec.ts
Comment (jordanpowell88) [inline angular-standalone/src/app/app.component.spec.ts:1]: I know these are existing but we should probably remove these tbh

**Review by jordanpowell88 — APPROVED** [review]
LGTM

File: angular-standalone/package.json (hunk 1 above)
Comment (AtofStryker) [reply angular-standalone/package.json:32]: I wont merge it in until signals is released. Then I will update the package.json and merge it in

File: angular-standalone/src/app/app.component.spec.ts
Comment (AtofStryker) [reply angular-standalone/src/app/app.component.spec.ts:1]: do you mean the existing component tests?

**Review by jennifer-shehane — APPROVED** [review]

**Comment by jennifer-shehane** [comment]
@AtofStryker Merged since signals released

**Pull request merged**